packages/lodestar/src/sync/utils/sync.ts

```diff
@@ -0,0 +1,83 @@
+import {IReputation, ReputationStore} from "../IReputation";
+import {BeaconBlock, BeaconBlockHeader, Checkpoint, Epoch, Slot} from "@chainsafe/eth2.0-types";
+import {IBeaconConfig} from "@chainsafe/eth2.0-config";
+import {hashTreeRoot} from "@chainsafe/ssz";
+import {blockToHeader} from "../../chain/stateTransition/util";
+import {IReqResp} from "../../network";
+
+export function isValidHeaderChain(config: IBeaconConfig, start: BeaconBlockHeader, blocks: BeaconBlock[]): boolean {
```

Comment:
I think we should give this name a slightly more descriptive name.
Also, consider adding an end param. (Then we could check the downloaded blocks against a known checkpoint root)
eg:
```typescript
/**
 * Ensure a chain of blocks links properly from start to end
 */
isValidChainOfBlocks(config: IBeaconConfig, start: Hash, end: Hash, blocks: BeaconBlock[]): boolean;
```

Reply:
again we don't have end hash at that point, only when we reach highest finalized epoch we can check if our checkpoint matches other peers checkpoint